drivers/mmc/core/sdio.c

```diff
@@ -1090,7 +1090,7 @@ static u32 mmc_select_low_voltage(struct mmc_host *host, u32 ocr)
 {
 	int bit;
 	u32 ocr_orig = ocr;
-#ifdef CONFIG_BCMDHD
+#if defined(CONFIG_BCMDHD) || defined(CONFIG_BRCMFMAC)
```

Comment:
Shouldn't BCMDHD ifdef removed?

Reply:
that's intentional, so that a switch back to bcmdhd is still possible